incubator/validator-core.spec/src/main/scripts/io/aklivity/zilla/specs/validator/core/schema/string.schema.patch.json

```diff
@@ -3,5 +3,40 @@
         "op": "add",
         "path": "/$defs/validator/types/enum/-",
         "value": "string"
+    },
+    {
+        "op": "add",
+        "path": "/$defs/validator/type/oneOf/-",
+        "value":
+        {
+            "if":
+            {
+                "properties":
+                {
+                    "type":
+                    {
+                        "const": "string"
+                    }
+                }
+            },
+            "then":
+            {
+                "properties":
+                {
+                    "type":
+                    {
+                        "const": "string"
+                    },
+                    "encoding":
+                    {
+                        "type": "string",
+                        "enum":
+                        [
+                            "utf_8"
```

Comment:
Also `utf_16` here?
Should it be `utf_8` or `utf-8`, etc?

Reply:
I was trying to align this with `StandardCharsets`. For example: `Charset UTF_8`